ocp_resources/multi_network_policy.py

```diff
@@ -0,0 +1,58 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class MultiNetworkPolicy(NamespacedResource):
+    """
+    MultiNetworkPolicy object.
+
+    API reference:
+    https://docs.openshift.com/container-platform/4.14/networking/multiple_networks/configuring-multi-network-policy.html
+    """
+
+    api_group = NamespacedResource.ApiGroup.K8S_CNI_CNCF_IO
+
+    def __init__(
+        self,
+        network_name=None,
+        policy_types=None,
+        ingress=None,
+        egress=None,
+        pod_selector=None,
+        **kwargs,
+    ):
+        """
+        Create and manage MultiNetworkPolicy
+
+        Args:
+            network_name (str): The name of the NetworkAttachmentDefinition that the policy will impact.
+            policy_types (list, optional): One or more of the valid ip policies.
+            ingress (list, optional): list containing a dictionary specifying the allowed "from" parameters.
+            egress (list, optional): list containing a dictionary specifying the allowed "to" parameters.
+            pod_selector (dict, optional): Map a label to match.
+        """
+        super().__init__(**kwargs)
+        self.network_name = network_name
+        self.policy_types = policy_types
+        self.pod_selector = pod_selector
+        self.ingress = ingress
+        self.egress = egress
+
+    def to_dict(self):
+        super().to_dict()
+        if not self.yaml_file:
+            if not self.network_name:
+                raise ValueError("network_name is required")
+            if self.pod_selector is None:
```

Comment:
In the docstring you have `optional`.
Can the user apply a YAML file without set pod selector?

I see in the doc that pod selector can be `podSelector:`

Reply:
Do you mean like in here?
apiVersion: k8s.cni.cncf.io/v1beta1
kind: MultiNetworkPolicy
metadata:
  name: deny-by-default
  annotations:
    k8s.v1.cni.cncf.io/policy-for: <network_name>
spec:
  podSelector:
  ingress: []

You cannot create it:
The MultiNetworkPolicy "deny-by-default" is invalid: spec.podSelector: Required value

Reply:
and if remove `podSelector` from the YAML file?

Reply:
Do you mean that if the user send a yaml, the code should make sure `podSelector` and 'network_name' are included? I don't see this is done in other resources' to_dict methods.

Reply:
> and if remove podSelector from the YAML file?
what happens if the user uses a yaml without `podSelector`? Does it work?

Reply:
It shouldn't work without this field. Pod selector must at least be an empty dict (what will affect all resources connected to the specified network).
There is a bug in the documentation...
https://issues.redhat.com/browse/OCPBUGS-29069

Reply:
In the bug the attribute exists without a value, what happens it you do not provide it?
```
apiVersion: k8s.cni.cncf.io/v1beta1
kind: MultiNetworkPolicy
metadata:
  name: deny-by-default
  annotations:
    k8s.v1.cni.cncf.io/policy-for: <network_name>
spec:
  ingress: []
```

Reply:
`The MultiNetworkPolicy "deny-by-default" is invalid: spec.podSelector: Required value`